test/inductor/test_pattern_matcher.py

```diff
@@ -1161,6 +1170,42 @@ def fn1(a, b):
         stable_topological_sort(graph)
         self.assertEqual(list(graph.nodes), [b, a, c])
 
+    def test_scaled_softmax(self):
+        def mul_softmax(a, b):
+            return F.softmax(a * b, dim=0)
+
+        def div_softmax(x, inv_scale):
+            return F.softmax(x / inv_scale, dim=0)
+
+        x = torch.randn(10, 10)
```

Comment:
Should we use in this test very large numbers, like the ones that were showing NaNs in the example that motivated all this?